index.html

```diff
@@ -44,6 +46,7 @@ <h2 class="meet-your-match">Meet your Match!</h2>
             Have a great idea for a product, but need help making it a reality?  We're here to help.  Startup Matchmaker is the best place for designers and developers to find each other.
           </p>
         <div class="create-profile">
+          <!-- Is there a reason you put a nav here instead of styling the div and link? -->
```

Comment:
I thought the nav was unnecessary, but put it there cuz I thought all links were navs.  Do you use navs for navigation bars only?